src/applications/disability-benefits/2346/components/PersonalInfoBox.jsx

```diff
@@ -0,0 +1,76 @@
+import moment from 'moment';
+import { genderLabels } from 'platform/static-data/labels';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+const PersonalInfoBox = ({
+  first,
+  last,
+  middle,
+  suffix,
+  gender,
+  dateOfBirth,
+  ssnLastFour,
+}) => {
+  const nameString = [first, middle, last, suffix]
```

Comment:
What do you think about changing this var name to `fullName`?

Reply:
works for me